Competition2018/src/main/java/competition/subsystems/gripperintake/commands/GripperViaTriggersCommand.java

```diff
@@ -26,8 +26,29 @@ public void initialize() {
 
     @Override
     public void execute() {
-        double intakePower = MathUtils.squareAndRetainSign(-oi.operatorGamepad.getLeftTrigger());
-        double ejectPower = MathUtils.squareAndRetainSign(oi.operatorGamepad.getRightTrigger());
+        double intakePower;
+        double ejectPower;
+        //double intakePower = MathUtils.squareAndRetainSign(-oi.operatorGamepad.getLeftTrigger());
+        //double ejectPower = MathUtils.squareAndRetainSign(oi.operatorGamepad.getRightTrigger());
+        if (oi.operatorGamepad.getLeftTrigger() >= 0.1) {
+            intakePower = (-oi.operatorGamepad.getLeftTrigger()) - 0.4;
```

Comment:
A good first attempt, but at 60% and above we just get saturated to 100%. What I'd suggest is a different approach:

J = "jump" value, the output value we jump up to (in your case, 0.4)
D = deadband size,where values are 1:1 (in your case, 0.1)

This means that from 0.1 to 1, we'd like to linearly go between 0.4 and 1.
That's an output change of 0.6 over an input change of 0.9, aka 0.6/0.9, aka 0.6666...

More generally that's (1-J) / (1-D).

Reply:
Additionally, all of this logic should be in a helper function, so you don't have it literally duplicated for left and right triggers.